ChangeLog.d/9382.txt

```diff
@@ -0,0 +1,4 @@
+API changes
```

Comment:
I think we should also mention (probably in a separate bullet point still in "API changes") the fact that the `cipher` module is no more able to dispatch to the `nist_kw` one.